### ISSUE TITLE
[Bug]: Multidimensional params for select with api request not working

### ISSUE THREAD

**Issue by Alph44rchitect** [tasteui/tasteui#563]
### What Happened

A multidimensional params array is not converted correctly.

The request url includes `&fields=[object Object]` instead of `&fields[users]=id,name`

### Component

Select Styled

### How to Reproduce

```blade
<x-select.styled :request="[
                    'url' => route('api.users'),
                    'params' => ['sort' => 'name', 'fields' => ['users' => 'id,name']],
                 ]" select="label:name|value:id" />
```

### Laravel Version

^11.10

### Livewire Version

^3.5

### TallStackUI Version

^1.29

### PHP Version

^8.2

### Operation System

Linux

**Comment by devajmeireles** [comment]
At first, I was reluctant not to do this, but adapting was not difficult at all. It's done! I'll launch it next week! 🥳

**Comment by devajmeireles** [comment]
Done! Released as part of v1.34.0: https://github.com/tallstackui/tallstackui/releases/tag/v1.34.0
